modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,274 @@
+        memory: "1Gi"
+        cpu: "4"
+        hugepages-1Gi: "4Gi"
+    command: ["sleep", "infinity"]
+  runtimeClassName: performance-cnf-performanceprofile <16>
+  volumes:
+  - name: hugepages
+    emptyDir:
+      medium: HugePages
+----
++
+--
+<1> Specify the same `target_namespace` where the `SriovNetwork` object is created. If you want to create the pod in a different namespace, change `target_namespace` in both the `Pod` spec and the `SriovNetwork` object.
+<2> Sets the group ownership of volume-mounted directories and files created in those volumes.
+<3> Controls which primary group ID the containers are run with.
+<4> Specify the DPDK image which includes your application and the DPDK library used by application.
+<5> Dropping all capabilities (`ALL`) from the container's securityContext means that the container will not have any special privileges beyond what is necessary for normal operation.
+<6> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access the Virtual Function I/O (VFIO) device.
+<7> Mellanox network interface controller (NIC) requires the `NET_RAW` capability.
+<8> Controls which user ID the container is run with.
+<9> This setting indicates that the container(s) within the pod should not be granted privileged access to the host system.
+<10>  This setting allows a container to escalate its privileges beyond the initial non-root privileges it may have been assigned.
+<11> This setting ensures that the container runs with a non-root user. This helps enforce the principle of least privilege, limiting the potential impact of a container compromise and reducing the attack surface.
+<12> Mount a hugepage volume to the DPDK pod under `/mnt/huge`. The hugepage volume is backed by the emptyDir volume type with the medium being `Hugepages`.
+<13> Optional: Specify the number of DPDK devices allocated to DPDK pod. This resource request and limit, if not explicitly specified, will be automatically added by the SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by the SR-IOV Operator. It is enabled by default and can be disabled by setting `enableInjector` option to `false` in the default `SriovOperatorConfig` CR.
+<14> Specify the number of CPUs. The DPDK pod usually requires exclusive CPUs to be allocated from the kubelet. This is achieved by setting CPU Manager policy to `static` and creating a pod with `Guaranteed` QoS.
+<15> Specify hugepage size `hugepages-1Gi` or `hugepages-2Mi` and the quantity of hugepages that will be allocated to the DPDK pod. Configure `2Mi` and `1Gi` hugepages separately. Configuring `1Gi` hugepage requires adding kernel arguments to Nodes. For example, adding kernel arguments `default_hugepagesz=1GB`, `hugepagesz=1G` and `hugepages=16` will result in `16*1Gi` hugepages be allocated during system boot.
```

Comment:
```suggestion
<15> Specify the size of the hugepage `hugepages-1Gi` or `hugepages-2Mi` and the number of hugepages to allocate to the DPDK pod. Configure `2Mi` and `1Gi` hugepages separately. Configuring the `1Gi` hugepage requires adding kernel arguments to nodes. For example, if you add the kernel arguments `default_hugepagesz=1GB`, `hugepages=1G`, and hugepages=16, `16*1Gi` hugepages are assigned at boot time.

Reply:
Going to leave as is to be consistent with other sections where same field is described.